impeller/tools/args.gni

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/common/config.gni")
+import("//flutter/testing/testing.gni")
+
+declare_args() {
+  impeller_debug =
+      flutter_runtime_mode == "debug" || flutter_runtime_mode == "profile"
+
+  # Whether the runtime capture/playback system is enabled.
+  impeller_capture = flutter_runtime_mode == "debug"
+
+  # Whether the Metal backend is enabled.
+  impeller_enable_metal = (is_mac || is_ios) && target_os != "fuchsia"
+
+  # Whether the OpenGLES backend is enabled.
+  impeller_enable_opengles = (is_linux || is_win || is_android || is_mac ||
+                              enable_unittests) && target_os != "fuchsia"
+
+  # Whether the Vulkan backend is enabled.
+  impeller_enable_vulkan = (is_linux || is_win || is_android || is_mac ||
+                            enable_unittests) && target_os != "fuchsia"
+
+  # Whether to use a prebuilt impellerc.
+  # If this is the empty string, impellerc will be built.
+  # If it is non-empty, it should be the absolute path to impellerc.
+  impeller_use_prebuilt_impellerc = ""
```

Comment:
This option is no longer being used since migrating to the engine_v2 recipes. Using it in the engine_v2 recipes would be difficult. If you're up for it, it can be safely deleted.

Reply:
Wilco in a followup. Is this also the case for `impeller_use_prebuilt_scenec`>